packages/webdriver/src/index.ts

```diff
@@ -21,8 +32,14 @@ export default class WebDriver {
          * for example). But only do this if the user has enabled this
          * behavior in the first place.
          */
-        if (params.enableDirectConnect) {
-            setupDirectConnect(params)
+        const { directConnectProtocol, directConnectHost, directConnectPort, directConnectPath } = params.capabilities as Options
```

Comment:
is it correct that both `params` and `params.capabilities` have `Options` type?

Reply:
I think they are defined in the `Options`